lib/util.js

```diff
@@ -199,6 +199,13 @@ function deprecate(text) {
   /* eslint-enable */
 }
 
+function deprecateWrap(text, fn, that) {
```

Comment:
seems we dont need the context `that` given `fn` has been bind to desired context?